contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -700,7 +956,7 @@ IECore::ConstObjectPtr AlembicScene::readAttribute( const Name &name, double tim
 
 void AlembicScene::writeAttribute( const Name &name, const IECore::Object *attribute, double time )
 {
-	throw IECore::NotImplementedException( "AlembicScene::writeAttribute" );
+	IECore::msg( IECore::Msg::Warning, "AlembicScene::writeAttribute", "Not implemented" );
```

Comment:
Yeah, I should probably implement that. I guess since we don't read them yet I thought it was OK not to include them for writing in this PR. Could I do this in a followup PR? I'd really like to get a Gaffer version out for testing before the IECoreImage shakeup gets merged...